massa-node/base_config/config.toml

```diff
@@ -279,6 +279,8 @@
     max_in_connections = 250
     # Cooldown before testing again old peer
     test_oldest_peer_cooldown = 720000
+    # Rate limitation on the data streams (per second)
+    rate_limit = 5_242_880    # 5 Mo / secs
```

Comment:
we should use a different limit for the bootstrap, probably something like 10_000_000